postman-collection/IoT Central.postman_collection.json

```diff
@@ -3958,6 +3958,242 @@
 				}
 			],
 			"description": "To learn more, see [How to use the IoT Central REST API to upload a file](https://docs.microsoft.com/azure/iot-central/core/howto-upload-file-rest-api)"
+		},
+		{
+			"name": "Manage dashboards",
+			"item": [
+				{
+					"name": "Add a dashboard - Line Chart",
```

Comment:
AS suggested in the article - can we have a single create dashboard that adds a dashboard with several example tiles?